extensions/warp-ipfs/src/store/files.rs

```diff
@@ -1275,6 +1275,9 @@ async fn _remove(ipfs: &Ipfs, root: &Directory, item: &Item) -> Result<(), Error
                     }
                 }
             }
+            if let Err(e) = root.remove_item(&directory.name()) {
+                tracing::error!(error = %e, item_name = %directory.name(), "unable to remove directory");
+            }
```

Comment:
It would be best in case something is inserted into that directory while we are scanning through and removing the items. A simple size check should do since empty directories dont count towards the size.